Library/Formula/sleuthkit.rb

```diff
@@ -1,17 +1,9 @@
 class Sleuthkit < Formula
   desc "Forensic toolkit"
   homepage "http://www.sleuthkit.org/"
-
-  stable do
-    url "https://downloads.sourceforge.net/project/sleuthkit/sleuthkit/4.1.3/sleuthkit-4.1.3.tar.gz"
-    sha256 "67f9d2a31a8884d58698d6122fc1a1bfa9bf238582bde2b49228ec9b899f0327"
-
-    # Upstream fix for https://github.com/sleuthkit/sleuthkit/issues/345
-    patch do
-      url "https://github.com/sleuthkit/sleuthkit/commit/39c62d6d169f8723c821ca7decdb8e124e126782.diff"
-      sha256 "f9419d6665a89df5625487dd50c16c12d1680477797917f8ec9182db55df4f7f"
-    end
-  end
+  url "https://github.com/sleuthkit/sleuthkit.git",
+    :tag => "sleuthkit-4.2.0",
```

Comment:
It is a Git checksum which means it's SHA-1 rather than SHA-256 (which is more secure). Additionally it's faster to `curl` a tarball than `git fetch` a repository.